ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/SubmissionValidator.scala

```diff
@@ -318,49 +315,26 @@ object SubmissionValidator {
       InputState,
   ) => LoggingContext => LogEntryAndState
 
-  def create[LogResult](
```

Comment:
I take it's not used anywhere (apart from tests)

Reply:
Exactly, yes.